test/periphery/integration/merkle-campaign/ll/claim/claim.t.sol

```diff
@@ -11,26 +12,56 @@ contract Claim_MerkleLL_Integration_Test is Claim_Integration_Test, MerkleLL_Int
         super.setUp();
     }
 
-    function test_ClaimLL() external givenCampaignNotExpired givenNotClaimed givenIncludedInMerkleTree {
+    function test_Claim_TimestampsInThePast()
```

Comment:
I refactored it to include the missing branches in https://github.com/sablier-labs/v2-core/pull/1027:

```tree
├── when scheduled start time zero
│   ├── when scheduled cliff duration zero
│   │   ├── it should create a stream with block.timestamp as start time
│   │   ├── it should create a stream with cliff as zero
│   │   └── it should emit a {Claim} event
│   └── when scheduled cliff duration not zero
│       ├── it should create a stream with block.timestamp as start time
│       ├── it should create a stream with cliff as start time + cliff duration
│       └── it should emit a {Claim} event
└── when scheduled start time not zero
    ├── it should create a stream with scheduled start time as start time
    └── it should emit a {Claim} event
```